mezzanine/core/admin.py

```diff
@@ -37,7 +37,6 @@ class BaseTranslationModelAdmin(TranslationAdmin):
         class Media:
             js = (
                 static("modeltranslation/js/force_jquery.js"),
-                static("mezzanine/js/%s" % settings.JQUERY_UI_FILENAME),
```

Comment:
Because I've had issues with jQuery UI I moved this into the grapelli-safe [base.html file](https://github.com/fermorltd/grappelli-safe/blob/2.2-compat/grappelli_safe/templates/admin/base.html#L20)